### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by danielcrk-cn** [chemerisuk/cordova-plugin-firebase-analytics#10]
Unfortunately, when adding the plugin, the build fails with this:

```
Updating ios build to use workspace.

Adding schemes

Error: EEXIST: file already exists, mkdir '/Users/myUserName/dev/myApp/platforms/ios/MyApp.xcworkspace/xcshareddata'
```

Any ideas? Really hope to get this working!
Thanks for all the work with the plugin!

**Comment by FdezRomero** [comment]
I had a similar warning. Are you using Cordova iOS 4.3.0 and a plugin with a Pod dependency (e.g. `phonegap-plugin-push@1.9.0`)?

I think the new native CocoaPods feature in Cordova conflicts with `cordova-plugin-cocoapods-support`, that this plugin uses.

**Comment by danielcrk-cn** [comment]
@FdezRomero Yeah, exactly, I am. Have you managed to get it working by downgrading the cordova version and the push plugin?

**Comment by danielcrk-cn** [comment]
However, i don't have cordova-plugin-cocoapod-support installed. Should I have?

**Comment by FdezRomero** [comment]
@danielcrk-cn This plugin installs `cordova-plugin-firebase-hooks`, which installs `cordova-plugin-cocoapod-support` for you. You have to manually install CocoaPods if you haven't though (at least in my case).

To fix the problem I manually added the pods needed by `phonegap-plugin-push` and this one in  `platforms/ios/Podfile` and installed again with `pod install`. I think I removed and re-added the iOS platform first.

**Comment by danielcrk-cn** [comment]
@FdezRomero Hmm, so basically, you run "cordova build ios" and get the error, manually copy the pod files and then run pod install?

**Comment by FdezRomero** [comment]
@danielcrk-cn No, there is a file actually called `Podfile` at `platform/ios/Podfile` that lists the pods that have to be installed. Check that you have all the pods that you need listed there. For example, this is my `Podfile` (you may have more pods if you have more plugins installed):

```
platform :ios, '7.0'
target 'NAME_OF_MY_APP' do
	pod 'GoogleCloudMessaging', '~> 1.2.0'
	pod 'GGLInstanceID', '~> 1.2.1'
	pod 'Firebase/Analytics'
end
```

Then running `pod install` will read the pods from your `Podfile` and install them if they are not yet. Then try to build again, and if if doesn't work, do `cordova platform rm ios` to remove the platform and `cordova platform add ios` to add it again. That's what I did and eventually everything worked.

**Comment by chemerisuk** [comment]
@FdezRomero 

> My understanding is that these problems would be solved with version 1.0.0 as it will use the native pods feature and ditch cordova-plugin-cocoapods-support
Correct, that's the plan. On the other case it would be nice to have a kind of polyfill that adds pods support for older versions of cordova. Could not find such polyfill yet.

**Comment by danielcrk-cn** [comment]
I really hate to be that guy, but is there any estimate on how long until version 1.0? Just so that I can manage some client expectations :)

**Comment by chemerisuk** [comment]
@danielcrk-cn sorry I have no idea. If you just need 4.3.0 support - feel free to create a fork. Then replace `<pod>` tags with `<framework>`. Should be pretty easy to do.

**Comment by xXcoronaXx** [comment]
i got this error, i've changed that file but now i have this error:

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/corona/Library/Developer/Xcode/DerivedData/DelSuper-ffswkjukwyjsglgmfzwyhrspednr/Build/Intermediates/DelSuper.build/Debug-iphonesimulator/DelSuper.build/Objects-normal/x86_64/PushPlugin.o DelSuper/Plugins/phonegap-plugin-push/PushPlugin.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/corona/delsuper_app_front_ionic_dev/platforms/ios/cordova/build-debug.xcconfig,-workspace,DelSuper.xcworkspace,-scheme,DelSuper,-configuration,Debug,-sdk,iphonesimulator,-destination,platform=iOS Simulator,name=iPhone 5s,build,CONFIGURATION_BUILD_DIR=/Users/corona/delsuper_app_front_ionic_dev/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/corona/delsuper_app_front_ionic_dev/platforms/ios/build/sharedpch

**Comment by chemerisuk** [comment]
Try 0.10.1

**Comment by asherwebb** [comment]
I had this same issue and this thread helped resolve the Error code 65 - the issue seems to be the `platforms/ios/Pod File` not having the firebase pod installed. I followed the suggestion from @FdezRomero - from my bash shell had to `cd/platforms/ios/` -> `pod install` and the problem was resolved. Note that the firebase entry was in the `Pod File` after installing `cordova-plugin-firebase-analytics`. The issue was it had not been installed.
